lti_consumer/static/css/student.css

```diff
@@ -0,0 +1,41 @@
+.xblock-student_view.xblock-student_view-lti_consumer h2.problem-header {
```

Comment:
Wouldn't that require us to make an assumption that sass is installed in the environment which is consuming this package? I'm not sure I would want to do that.